src/crystal/lib_iconv.cr

```diff
@@ -4,6 +4,11 @@ require "c/stddef"
   {% raise "The `without_iconv` flag is preventing you to use the LibIconv module" %}
 {% end %}
 
+# Supported library versions:
+#
+# * libiconv-gnu
```

Comment:
Don't we use any iconv library provided by the system? It's a POSIX standard.

Reply:
Naming is a bit confusing unfortunately. But AFAIU the POSIX standard is just called `iconv` without `lib`. We have bindings for that as part of `LibC`.
`libiconv` as an explicit library is only used on Windows and Android (< API 28), or when explicitly asked with `-Duse_libiconv`. And then we only use the GNU variant of the library.

https://github.com/crystal-lang/crystal/blob/401eb47bf373f11b2da6e382c306ead36615e1d6/src/crystal/iconv.cr#L1-L7